DFC.ServiceTaxonomy.Taxonomies/Controllers/AdminController.cs

```diff
@@ -105,6 +107,11 @@ public async Task<IActionResult> CreatePost(string id, string taxonomyContentIte
 
             dynamic model = await _contentItemDisplayManager.UpdateEditorAsync(contentItem, _updateModelAccessor.ModelUpdater, true);
 
+            if (!await ValidateTaxonomyTermURL(contentItem, taxonomy))
```

Comment:
We should keep the taxonomy module generic, so that we can reuse it.
Perhaps add plug-in custom validation support or something of that ilk.